Dockerfile.debug

```diff
@@ -0,0 +1,27 @@
+FROM golang:latest as build
+WORKDIR /go/src/github.com/pomerium/pomerium
+
+RUN apt-get update \
+    && apt-get -y install zip
+
+# cache depedency downloads
+COPY go.mod go.sum ./
+RUN go mod download
+COPY . .
+
+# build
+RUN make build-deps
+RUN make build-debug NAME=pomerium
+RUN make build-debug NAME=pomerium-cli
+RUN touch /config.yaml
+RUN go get github.com/go-delve/delve/cmd/dlv
+
+FROM alpine:latest
```

Comment:
The distroless environment is limiting and has some weird changes like having `sh` live under `/busybox`.  If someone is really getting into the weeds, alpine with additional tools available can be helpful.